src/touchable.tsx

```diff
@@ -123,6 +126,16 @@ export interface PropsMaybeOnPress extends CommonProps {
 
 type Props = PropsHref | PropsTo | PropsOnPress | PropsMaybeHref | PropsMaybeTo | PropsMaybeOnPress;
 
+const getPrefixedDataAttributes = (attrs?: DataAttributes) => {
```

Comment:
this could be moved to utils/dom.tsx if needed by other components following this convention